frontend/src/components/EntityBulkAdd.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div v-for="num in entityFormComponentsCountRange" :key="`ef-${num}`">
+    <EntityForm :ref="`entity-form-${num}`" :schema="schema"
```

Comment:
I am missing some sort of a remove/cross element to remove a form if I over-click or I just changed my mind and want to remove an entity from the bulk add. Not a big deal, but I've been intuitively looking for it.